src/api/stats_api.js

```diff
@@ -758,7 +758,7 @@ module.exports = {
                     type: 'array',
                     items: {
                         type: 'object',
-                        required: ['bucket_name', 'quota_size_precent', 'quota_quantity_percent', 'capacity_precent', 'is_healthy', 'tagging'],
+                        required: ['bucket_name', 'quota_size_precent', 'quota_quantity_percent', 'capacity_precent', 'is_healthy', 'tagging', 'objectbucket_used_bytes'],
```

Comment:
I think that we don't need to state that it is `objectbucket` (similar to `ocs_objectbucket_used_bytes` for RADOS S3), I think we can simply call it `bucket_used_bytes` as we have `bucket_capacity` (which weirdly presents the percentage), `bucket_size_quota`, `bucket_tagging`, ect'
```suggestion
                        required: ['bucket_name', 'quota_size_precent', 'quota_quantity_percent', 'capacity_precent', 'is_healthy', 'tagging', 'bucket_used_bytes'],
```

WDYT?

Reply:
I am fine with this, I gave this name to align with OCS metrics name mentioned in BZ
 @dannyzaken